packages/react-instantsearch-hooks-web/src/widgets/SearchBox.tsx

```diff
@@ -19,10 +19,11 @@ type UiProps = Pick<
 export type SearchBoxProps = Omit<SearchBoxUiComponentProps, keyof UiProps> &
   UseSearchBoxProps;
 
-export function SearchBox(props: SearchBoxProps) {
-  const { query, refine, isSearchStalled } = useSearchBox(props, {
-    $$widgetType: 'ais.searchBox',
-  });
+export function SearchBox({ queryHook, ...props }: SearchBoxProps) {
+  const { query, refine, isSearchStalled } = useSearchBox(
```

Comment:
do you not need to pass the `props` to this hook as well as the `queryHook`?

Reply:
only the connector props are needed here